src/AlgebraicGeometry/Schemes/Covering/Objects/Methods.jl

```diff
@@ -68,6 +69,25 @@ function update_glueing_graph(C::Covering)
   return gg
 end
 
+## prune the covering by throwing away empty charts and then compute
+## the glueing graph afterwards 
+## (relevant data for connectedness of glueing)
+function pruned_glueing_graph(C::Covering)
+  v = findall(U->!is_empty(U), C.patches)
+  m = length(v)
+  gt = Graph{Undirected}(m)
+  for (X, Y) in keys(glueings(C))
+    i = findfirst(==(C[X]),v)
```

Comment:
```suggestion
    i = findfirst(===(C[X]),v)
```
? so that no expensive computation is triggered? not sure if this could trigger one or not.

Reply:
C[X] is an int64.

Reply:
okay.